src/main/java/com/example/DoughnutShop.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+public class DoughnutShop {
+
+    public String prepareDoughnut(String doughnutType) {
+        String preparedDoughnut;
+
+        if (doughnutType.equals("Chocolate")) {
```

Comment:
Can use constants (e.g., `String CHOCOLATE = "Chocolate"`) for all `doughtnutType` checks